src/app/server/Server.cpp

```diff
@@ -297,6 +301,41 @@ CHIP_ERROR Server::Init(const ServerInitParams & initParams)
     return err;
 }
 
+CHIP_ERROR Server::InitFailSafe()
```

Comment:
nit, is there any point to this returning `CHIP_ERROR` instead of void? Right now it looks like you always return `CHIP_NO_ERROR`

Reply:
Good catch